gradle-metric-schema/src/main/java/com/palantir/metric/schema/gradle/GenerateMetricSchemaTask.java

```diff
@@ -39,6 +44,14 @@ public GenerateMetricSchemaTask() {
         libraryName.set(defaultLibraryName());
     }
 
+    @Override
+    @InputFiles
+    @SkipWhenEmpty
+    @PathSensitive(PathSensitivity.RELATIVE)
+    public final FileTree getSource() {
+        return super.getSource();
+    }
```

Comment:
this is what Gradle's SourceTask has baked in:

```

    @InputFiles
    @SkipWhenEmpty
    @PathSensitive(PathSensitivity.ABSOLUTE)
    public FileTree getSource() {
        return this.sourceFiles.getAsFileTree().matching(this.patternSet);
    }
```

Reply:
yup, absolute is the safe thing to do but it's unnecessary in sane environments